### PR TITLE
Fix cross_section_ppi in util.py for wradlib 2.0

### PR DESCRIPTION
**Pull request by JulianGiles**
Fixed cross_section_ppi  to work with wradlib 2.0

### CODE REVIEW

File: wradlib/tests/test_util.py [hunk 1]
```diff
@@ -326,3 +327,55 @@ def test_find_bbox_indices(bb_data):
 
     bbind = util.find_bbox_indices(bb_data.grid, bb_data.inside2)
     assert np.array_equal(bbind, [1, 1, 4, 7])
+
+
+@requires_data
```
Comment (kmuehlbauer) [inline wradlib/tests/test_util.py:332]: @requires_gdal is needed here, too

File: wradlib/util.py [hunk 2]
```diff
@@ -1200,14 +1162,23 @@ def cross_section_ppi(
     method = kwargs.get("method", None)
     tolerance = kwargs.get("tolerance", None)
 
+    # Deprecation warning
+    real_beams = kwargs.get("real_beams")
```
Comment (kmuehlbauer) [inline wradlib/util.py:1166]: Just remove this, if it was removed in 2.0. Better a clean cut now.

**Comment by kmuehlbauer** [comment]
@JulianGiles Just one minor thing, otherwise LGTM! Thanks!

File: wradlib/util.py (hunk 2 above)
Comment (JulianGiles) [reply wradlib/util.py]: ok, done

**Comment by kmuehlbauer** [comment]
@JulianGiles Sorry, I forgot, you might add an entry to https://github.com/wradlib/wradlib/blob/main/docs/release_notes.md.

**Comment by kmuehlbauer** [comment]
Sorry for the delay @JulianGiles. Thanks for taking care.

**Pull request merged**